.circleci/config.yml

```diff
@@ -170,13 +194,20 @@ jobs:
 
   ios-integration-test:
     description: "Run iOS integration tests for Cordova"
+    resource_class: macos.x86.medium.gen2
```

Comment:
Hmm interesting why this is still using x86 architecture... I tried looking in CircleCI docs and couldn't find a different machine, so I think we can go with this.

Reply:
I copied this from purchases-ios